### PR TITLE
CONSOLE-3246: Update i18next to version 21.x.x

### PR DESCRIPTION
**Pull request by cyril-ui-developer**
### Before
Screen Shot 2022-08-19 at 8 34 46 AM

Screen Shot 2022-08-19 at 8 36 22 AM

### After
Screen Shot 2022-08-1
![Screen Shot 2022-08-19 at 8 34 46 AM

Screen Shot 2022-08-19 at 8 36 22 AM

### CODE REVIEW

File: frontend/packages/operator-lifecycle-manager/locales/en/olm.json [hunk 1]
```diff
@@ -180,8 +180,8 @@
   "Change Subscription update channel": "Change Subscription update channel",
   "Which channel is used to receive updates?": "Which channel is used to receive updates?",
   "Operator <1>{{name}}</1> will be removed from <3>{{namespace}}</3>. If your Operator configured off-cluster resources, these will continue to run and require manual cleanup.": "Operator <1>{{name}}</1> will be removed from <3>{{namespace}}</3>. If your Operator configured off-cluster resources, these will continue to run and require manual cleanup.",
-  "The console plugin provided by this operator will be disabled and removed.": "The console plugin provided by this operator will be disabled and removed.",
-  "The console plugin provided by this operator will be disabled and removed._plural": "The console plugins provided by this operator will be disabled and removed.",
+  "The console plugin provided by this operator will be disabled and removed._one": "The console plugin provided by this operator will be disabled and removed.",
+  "The console plugin provided by this operator will be disabled and removed._other": "The console plugin provided by this operator will be disabled and removed.s",
```
Comment (cyril-ui-developer) [inline frontend/packages/operator-lifecycle-manager/locales/en/olm.json:184]: Seems to me that pluralization for words before the last word in a sentence or phrase were manually updated in locales json. Wanted to check if my assumption is correct. Does anyone know?

File: frontend/packages/kubevirt-plugin/locales/en/kubevirt-plugin.json [hunk 2]
```diff
@@ -623,12 +623,12 @@
...
-  "The following disk will not be included in the snapshot_plural": "The following disk will not be included in the snapshots",
-  "Edit the disk or contact your cluster admin for further details.": "Edit the disk or contact your cluster admin for further details.",
-  "Edit the disk or contact your cluster admin for further details._plural": "Edit the disks or contact your cluster admin for further details.",
+  "Disks included in this snapshot ({{count}})_one": "Disks included in this snapshot ({{count}})",
+  "Disks included in this snapshot ({{count}})_other": "Disks included in this snapshot ({{count}})s",
+  "The following disk will not be included in the snapshot_one": "The following disk will not be included in the snapshot",
+  "The following disk will not be included in the snapshot_other": "The following disk will not be included in the snapshots",
+  "Edit the disk or contact your cluster admin for further details._one": "Edit the disk or contact your cluster admin for further details.",
+  "Edit the disk or contact your cluster admin for further details._other": "Edit the disk or contact your cluster admin for further details.s",
```
Comment (jhadvig) [inline frontend/packages/kubevirt-plugin/locales/en/kubevirt-plugin.json:631]: The `s` after the sentence looks wrong

File: frontend/packages/operator-lifecycle-manager/locales/en/olm.json (hunk 1 above)
Comment (jhadvig) [reply frontend/packages/operator-lifecycle-manager/locales/en/olm.json:184]: Here the `s` character appears as well

**Review by sg00dwin — APPROVED** [review]

**Comment by TheRealJon** [comment]
e2e tests are hitting a runtime error on the edit webhook secret page:

```
4) Interacting with the create secret forms
   5) Webhook secret
      ✔ creates webhook secret
      ✔ check for created webhook secret value
A Jasmine spec timed out. Resetting the WebDriver Control Flow.
A Jasmine spec timed out. Resetting the WebDriver Control Flow.
A Jasmine spec timed out. Resetting the WebDriver Control Flow.
      ✖ edits webhook secret (3 failures)
      • check for edited webhook secret value
      • deletes the webhook secret
```
 
image

**Comment by cyril-ui-developer** [comment]
@sg00dwin Thanks.

**Comment by cyril-ui-developer** [comment]
@TheRealJon Just seeing a couple failing tests now. Will investigate and fix asap. 
Thanks.

**Comment by spadgett** [comment]
@cyril-ui-developer Have we decided how plugins will support n-1 compatibility? Are we asking them to take approach (2) in the spike? https://issues.redhat.com/browse/CONSOLE-3175

Let's raise this with the plugin teams to see if how much of a burden that is. Maybe we can provide a script that will automatically add both formats to their catalogs. We could still explore the possibility of the backend doing it automatically.

cc @KevinFCormier

**Comment by spadgett** [comment]
/hold

Let's hold until we have agreement on the plugin issue.

**Comment by cyril-ui-developer** [comment]
Follow-on PR : https://github.com/openshift/console/pull/12004

**Comment by cyril-ui-developer** [comment]
Closing this PR. See the follow-on[ PR](https://github.com/openshift/console/pull/12124) for the updated changes.

**Pull request closed without merging**